feat: add `blas/base/wasm/sscal` (#5224)
stdlib-js/stdlib: lib/node_modules/@stdlib/blas/base/wasm/sscal/docs/repl.txt

{{alias}}.main( N, alpha, x, strideX )
    Multiplies a vector `x` by a constant `alpha`.

>     The `N` and stride parameters determine which elements in the strided arrays

Reviewer: ```suggestion
    The `N` and stride parameters determine which elements in the strided array
```

    alpha: number
        Constant.

    x: Float32Array
        Input array.

    strideX: integer
        Index increment for `x`.

    Returns
    -------
    x: Float32Array
        Input array.

    Examples
    --------
    // Standard usage:
    > var x = new {{alias:@stdlib/array/float32}}( [ -2.0, 1.0, 3.0, -5.0 ] );
    > {{alias}}.main( x.length, 5.0, x, 1 )
    <Float32Array>[ -10.0, 5.0, 15.0, -25.0 ]

    // Using `N` and stride parameters:
    > x = new {{alias:@stdlib/array/float32}}( [ -2.0, 1.0, 3.0, -5.0 ] );
    > {{alias}}.main( 2, 5.0, x, 2 )
    <Float32Array>[ -10.0, 1.0, 15.0, -5.0 ]

    // Using view offsets:
    > var x0 = new {{alias:@stdlib/array/float32}}( [ 1.0, -2.0, 3.0, -4.0 ] );
    > var x1 = new {{alias:@stdlib/array/float32}}( x0.buffer, x0.BYTES_PER_ELEMENT*1 );
    > {{alias}}.main( 2, 5.0, x1, 2 )
    <Float32Array>[ -10.0, 3.0, -20.0 ]
    > x0
    <Float32Array>[ 1.0, -10.0, 3.0, -20.0 ]

{{alias}}.ndarray( N, alpha, x, strideX, offsetX )
    Multiplies a vector `x` by a constant `alpha` using alternative indexing
    semantics.

    While typed array views mandate a view offset based on the underlying

>     buffer, the offset parameters support indexing semantics based on starting

Reviewer: ```suggestion
    buffer, the offset parameter supports indexing semantics based on a starting
```

>     indices.

Reviewer: ```suggestion
    index.
```
